backend/inspirehep/search/facets.py

```diff
@@ -283,13 +293,14 @@ def flattened_refereed = [];
                                     ],
                                 }
                             },
-                            "citations_count": {"sum": {"field": "citation_count"}},
-                            "average_citations": {"avg": {"field": "citation_count"}},
+                            "citations_count": {"sum": {"field": field}},
+                            "average_citations": {"avg": {"field": field}},
                         },
                     },
                 },
             }
         },
+        "excludes-self-citations": "false" if field == "citation_count" else "false",
```

Comment:
is this supposed to be in reaction to my suggestion? I don't think this is in the right place (plus it's always `"false"`), and anyway you're probably right that it's not really needed.

Reply:
Argh, this is mistake. It was just a test and I didn't want to put it there. Sorry.